pkgs/applications/audio/i-score/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  boost,
+  cln,
+  cmake,
+  fetchgit,
+  gcc5,
+  ginac,
+  jamomacore,
+  kde5,
+  libsndfile,
+  ninja,
+  portaudio,
+  qtbase,
+  qtdeclarative,
+  qtimageformats,
+  qtsvg,
+  qttools,
+  qtwebsockets,
+  rtaudio,
+  stdenv
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.0.0-a67";
+  name = "i-score-v${version}";
```

Comment:
This is equivalent to having "v" in the version; afaik, the tools only look at `name` and extract the version via parsing, not by looking at the `version` attribute.